py2plantuml.sh

```diff
@@ -17,16 +17,70 @@ create_svg_files() {
 
 from_dir=$1
 out_dir=$2
-shift
-shift
-plantuml=/opt/homebrew/bin/plantuml
 python=python3
+plantuml=/opt/homebrew/bin/plantuml
+svg_dep=secure
+function usage(){
+    echo "$(basename $0) [-i | --from_dir ] [ -o | --out_dir] [-d | --dependency_install]  [-p | --plantweb_dep_install] [ --skip_uses_relation ] [ --info ] [ --debug ] [ --trace ] [-h | --help]"
+}
+
+while [[ "$1" != "" ]]; do
+    case $1 in
+        -i | --from_dir )
+        from_dir=$2;
+        shift;
+        ;;
+        -o | --out_dir )
+        out_dir=$2;
+        shift;
+        ;;
+        -d | --dependency_install )
+        echo "Installing plantuml dependency for rendering SVC....."
+        if java -version 2>&1 >/dev/null | grep -E "\S+\s+version" ; then
+          wget -O plantuml.jar https://github.com/plantuml/plantuml/releases/download/v1.2022.13/plantuml.jar
+          echo "Finished installing plantuml dependency for the script!!"
+        else
+          echo "Java is needed for using plantuml dependency...Please install Java first" && exit 1
+        fi
```

Comment:
Thanks @MSarfraz-CR  this is very good, maybe inform as well that Graphviz is needed (if possible check it is present but this is not mandatory).